steps/docker.py

```diff
@@ -82,3 +82,15 @@ def step_impl(context, dockerfile, tag=''):
         module_args += ' -t %s' % tag
     assert context.remote_cmd('command',
                                module_args='%s %s' % (module_args, dockerfile))
+
+@when('docker remove all of images')
+def step_impl(context):
+   '''Remove all of images'''
+   assert context.remote_cmd('shell',
+                             module_args='docker rmi `docker images -a -q`')
```

Comment:
Won't this fail if there are no images present? Maybe use `xargs -r` instead? E.g. `docker images -a -q | xargs -r docker rmi`. The `-r` is nifty:

```
 -r, --no-run-if-empty
        If  the standard input does not contain any nonblanks, do not run the command.  Normally, the
        command is run once even if there is no input.  This option is a GNU extension.
```